reana_job_controller/config.py

```diff
@@ -31,3 +31,30 @@
 
 MULTIPLE_COMPUTE_BACKENDS = os.getenv('MULTIPLE_COMPUTE_BACKENDS', False)
 """Allow multiple job compute backends."""
+
+JOB_HOSTPATH_MOUNTS = [
+    ('foo', '/foo'),
+]
+"""List tuples composed of name and path to create hostPath's inside jobs.
+
+This configuration should be used only when one knows for sure that the
+specified locations exist in the cluster nodes.
+
+For example, if you are running REANA on Minikube with a single VM you would
+have to mount in Minikube the volume you want to be attached every job:
+
+.. code-block::
+
+    $ minikube mount /usr/mydata:/mydata
```

Comment:
Just `/mydata` or else `/opt/mydata` (or else `/usr/local/share/mydata`) may be better example that `/usr/mydata`.